scc-hypervisor-collector/xml/art-scc-hypervisor-collector.xml

```diff
@@ -0,0 +1,324 @@
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+  <!ENTITY productnamex "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productname;</phrase><phrase xmlns='http://docbook.org/ns/docbook' os='article'>&sls;</phrase>">
+   <!ENTITY productnumberx "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productnumber;</phrase>">
+]>
+<article xml:id="article-scc-hypervisor-collector"
+         xmlns="http://docbook.org/ns/docbook"
+         version="5.0"
+         xml:lang="en"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Hypervisor details in &scc;</title>
+  <titleabbrev>Hypervisor details in &scc;</titleabbrev>
+  <info><productname>&productnamex;</productname>
+    <productnumber>&productnumberx;</productnumber><date>
+<?dbtimestamp format="B d, Y" ?></date>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker>
+        <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+        <dm:component>Documentation</dm:component>
+        <dm:assignee>tbazant@suse.com</dm:assignee>
+      </dm:bugtracker>
+      <dm:editurl>https://github.com/SUSE/doc-unversioned/blob/main/xml/</dm:editurl>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+    <abstract>
+      <para>
+        This article describes how to provide customers with details about the
+        hypervisors they use. The collected data is available under the
+        customers' &scc; (SCC) account.
+      </para>
+    </abstract>
+  </info>
+  <sect1 xml:id="scc-hypervisor-collector-whatis">
+    <title>What is &scchvc;?</title>
+
+    <para>
+      &scchvc; is a command line tool that collects details about customer
+      hypervisor solutions and uploads them to SCC.
+    </para>
+  </sect1>
+  <sect1 xml:id="scc-hypervisor-collector-requirements">
+    <title>Requirements</title>
+
+    <para>
+      To collect hypervisor data and upload them to SCC, you need to:
+    </para>
+
+    <itemizedlist>
+      <listitem>
+        <para>
+          Run a supported and registered SLE-based host with all update patches
+          applied.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configure a &vmhost; with a supported hypervisor that hosts one or
+          more supported &vmguest;s.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Install the <package>scc-hypervisor-collector</package> package and
+          configure the collector to match your environment.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+  <sect1 xml:id="scc-hypervisor-collector-how-it-works">
+    <title>How does &scchvc; work?</title>
+
+    <para>
+      The
+      <systemitem class="daemon">scc-hypervisor-collector.timer</systemitem>
```

Comment:
I would recommend structuring this along the lines of manually configuring the command to run via the scchvc user account first, then validate that it runs correctly when triggered by the systemd service, and only when that is working should there be mention of the systemd timer.

That may mean swapping section 3 and 4?